.github/workflows/test.yml

```diff
@@ -0,0 +1,14 @@
+name: Go Toolchain Tests
+on:
+  pull_request:
+  push:
+    branches:
+      - cf
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run tests
```

Comment:
```suggestion
      - name: Build Go and run tests
```

Reply:
I'll consider building as part of the test :)

Reply:
lol, ok